app/src/ingestion/elements.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import List
+
+
+class TextType(Enum):
+    NARRATIVE_TEXT = "NarrativeText"
+    LIST_ITEM = "ListItem"
+    LIST = "List"
+
+
+@dataclass
+class Page:
+    # The actual page in the document
+    pdf_page_number: int
+
+    # The label of the page in the document
+    # E.g., "i", "ii", "1", "2", etc.
+    document_page_number: str
+
+
+@dataclass
+class Heading:
+    title: str
+    level: int
+    page: Page
```

Comment:
I only use an `int` page number: https://github.com/navapbc/labs-decision-support-tool/pull/57/files#diff-aecde49f56fa9585be76a3e8af06031a4037575977c2030c02052a7a9af2c34eR16

Having to populate `Page.document_page_number` involves parsing the pdf text.

We'll have to resolve the 2 `Heading` classes.

Reply:
Left a related comment here -- https://github.com/navapbc/labs-decision-support-tool/pull/57#discussion_r1729203838 I'll follow your lead here, let me know how you'd like me to sync these up and I'll make the change in this PR before merging.